idd/Energy+.idd.in

```diff
@@ -16581,58 +16592,50 @@ SurfaceProperty:ExposedFoundationPerimeter,
        \units dimensionless
        \minimum 0.0
        \maximum 1.0
-       \default 1.0
```

Comment:
Should the `\default` be removed? If so, then this should probably be `\required-field`.